### PR TITLE
Simplify carousel tutorial, mostly structure page

### PR DESCRIPTION
**Pull request by yatil**
### _**This pull request was replaced by  #365 as I made an error that made this PR hard to review.**_

#365 is equal to this pull request.

### CODE REVIEW

**Comment by yatil** [comment]
**Currently “Approved Draft”:** https://www.w3.org/WAI/tutorials/carousels/structure/
**Changes from this PR:** https://w3c.github.io/wai-tutorials/carousels/structure/

You can comment and/or approve the changes on this page with the diff of the file(s): https://github.com/w3c/wai-tutorials/pull/317/files

**Note:** Use the blue [+] button to comment on specific lines of the page. You can then use the top-right green button to just comment or request changes. Alternatively you are also able to outright approve the changes there. You can find more information about [Pull Request Reviews in Github’s help](https://help.github.com/articles/about-pull-request-reviews/).

**Comment by yatil** [comment]
*sigh* I made a change and it messed up the changed files tab. I try to correct that. Sorry.

**Comment by yatil** [comment]
### _**This pull request was replaced by  #365 as I made an error that made this PR hard to review.**_

#365 is equal to this pull request.

Sorry again.

File: source/carousels/animations.html.erb.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ Provide a button to allow users to stop and resume the animations. The example b
 <%= notes_start %>
 {:/nomarkdown}
 
-**Note:** The script only replaces the value of the button and its attributes.Replace the button entirely would result in a lost keyboard focus.
+**Note:** The script only replaces the value of the button and its attributes, rather than to replace the button entirely. If the button is removed, the focus would be lost for keyboard users.
```
Comment (susanatx) [inline source/carousels/animations.html.erb.md]: Cutting/changing out words at editor's discretion: 

**Note:** The script replaces the value of the button and its attributes; it does not entirely replace it. If the button is removed, the focus will be lost for keyboard users.

File: source/carousels/animations.html.erb.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ Provide a button to allow users to stop and resume the animations. The example b
...
 
 {::nomarkdown}
 <%= notes_end %>
 {:/nomarkdown}
 
 ## Pause on hover and focus
 
-Another useful approach is to pause the carousel animation when the carousel receives keyboard focus or is hovered over by mouse. Pausing the animation ensures that keyboard users do not lose focus. It is also useful for people who need more time to read the content and to direct the mouse cursor to a link or control inside the carousel item.
+Another useful approach is to pause the carousel animation when the carousel receives focus by keyboard or is hovered over by mouse. Pausing the animation ensures that keyboard users do not lose focus. It is also useful for people who need more time to read the content and to direct the mouse cursor to a link or control inside the carousel item.
```
Comment (susanatx) [inline source/carousels/animations.html.erb.md]: "Another approach is to pause the animation when the carousel receives focus"

Taking out unneeded "carousel" and the word "useful." (We wouldn't suggest this method if it were useless.)

File: source/carousels/index.html.erb.md [hunk 3]
```diff
@@ -12,21 +12,21 @@ wcag_success_criteria:
...
 
-- **[Full Content](examples/carousel.html):** The complete example source code.
+See also a [Complete Example](full-code.html) including the full source code.
 
 ## Why is this important?
 
-Carousels are frequently used, in particular on home pages, to draw readers' attention to featured articles and products of the website. Given their prominence, inaccessible carousels can be a major obstacle for many website visitors. On the other hand, accessible carousels provide more effective access for many users including:
+Carousels, also referred to as “slide shows” and “sliders”, display one item from a series of content items. Common examples are scrolling news headlines and images, like featured articles. Carousels often use animations to move from item to item, which can be distracting for some users. They may also move too fast for some users to be able to read.
+
+Carousels should have functionality to allow users to pause and advance animations. They also need to be implemented in a way that is usable by keyboard. Given their prominence, accessible carousels are essential for many website visitors including:
```
Comment (susanatx) [inline source/carousels/index.html.erb.md]: Because they are so common on websites, accessible carousels are essential for many users including:

**Pull request closed without merging**